### DISCUSSION TITLE
Welcome to waipy Discussions!

### DISCUSSION THREAD

**Discussion by mabelcalim** [mabelcalim/waipy#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Manuela-RT** [comment]
Hi, I am relatively new with Python and I began to explore the Waipy package for the analysis of vegetation and climate data in the Amazon basin. I already have my first plots, but I was wondering if it is possible to increase the font size in the axes and the title?

Thanks!
